src/internal/_assocPath.js

```diff
@@ -0,0 +1,11 @@
+var _assoc = require('./_assoc');
+var _slice = require('./_slice');
+
+
+module.exports = function _assocPath(path, val, obj) {
```

Comment:
We should also consider the current [behaviour][1] of `R.path`:

> ```javascript
> assert.strictEqual(R.path('', obj), undefined);
> ```


[1]: https://github.com/ramda/ramda/blob/v0.10.0/test/path.js#L25